src/test/test-fs-util.c

```diff
@@ -864,14 +864,16 @@ static void test_path_is_encrypted_one(const char *p, int expect) {
 }
 
 static void test_path_is_encrypted(void) {
-        log_info("/* %s */", __func__);
+        int booted = sd_booted();
```

Comment:
maybe add comment here? ("If this is run in build environments such as koji, /dev might be a reguar fs, let's protect against that?")